pkg/plugin/metrics.go

```diff
@@ -22,4 +23,18 @@ var (
 			"operation",
 		},
 	)
+
+	kmsLatencyMetric = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "aws_encryption_provider_kms_operation_latency_seconds",
+			Help: "Response latency in seconds for aws encryption provider kms operation ",
+			Buckets: []float64{0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0,
```

Comment:
can we start it from 0.25? u think that will work?

Reply:
so i guess buckets would be:

0.25, 0.5, 1, 2, 4, 8.....

Reply:
i think I've seen the response time to be 5-6ms but not in fraction of ms. so lets stick to ms. 

also... looking at https://play.golang.org/p/nRngWZmTVQG , i think it may make sense to start with 10ms, factor of 2, and 12 buckets (max of ~20s)

please suggest

Reply:
I've pushed the code with prometheus.ExponentialBuckets(10, 2, 12) which should create following buckets:

[10 20 40 80 160 320 640 1280 2560 5120 10240 20480]

pls let me know if that sounds ok

Reply:
sounds good. thanks.